src/redux/features/books/booksSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    AddBook: (state, { payload }) => {
+      state.books.push({ payload });
+    },
+    RemoveBook: (state, action) => {
+      const itemId = action.payload;
+      state.books.push(state.books.filter((item) => item.id !== itemId));
```

Comment:
- [ OPTIONAL ] I think the previous code reviewer doesn't mention that but still very important to do it, so I think it would be better if you just made the filter by id here without pushing each time since we want to remove the book not to push then filter because you already pushed in addbook above 💯🤩🔥

### It would be somthing like this
> state.books = state.books.filter((book) => book.id !== action.id)